content/2017.03.building-projects-as-a-way-to-learn.rst

```diff
@@ -0,0 +1,258 @@
+repository somewhere forever.
+
+Writing software to solve problems is not enough, to keep motivated on
+the long term. You need a greater goal than just writing code.
+
+Knowing that, how do you keep motivated building things that are
+thrown away?
+
+Different people or communities might have different guiding questions.
+
+For us the question is: As well as solving the problem, what can we
+learn while building this solution that can be useful for us and for
+our community to help us go faster and build more robust software
+later.
+
+For Mozilla the question is: How do I improve Firefox
+so that people want to use it as their main browser?
+
+Both questions lead to goals that we set to ourselves. We are using
+software development to reach them.
+
+Mozilla is calling that the True North, we are calling it a vision,
+some other people might call it a utopia.
+
+In both cases that vision should set the route of our ambition and
+point out a direction for everyone to sail to.
+
+How do you set course for a vision?
+===================================
+
+.. image:: {filename}/images/draken.jpg
+    :alt: Draken sailing to the North
+    :align: center
+
+
+|OKRs| are a way to cut your vision in steps and to move your vision
+forward.
+
+To be able to define OKRs from your vision, you need to be able to find
+the fastest way to get closer to your aim, however what we can learn
+from sailing is that the fastest way is rarely the shortest.
+
+At Mozilla, we have a vision that is written down in the
+`Mozilla Manifesto <https://www.mozilla.org/en-US/about/manifesto/>`_.
+
+Mitchell Baker does a great job making sure everyone at Mozilla stays
+aligned with that vision.
+
+However agreeing on the fastest way and turning the Mozilla vision
+into OKRs is really challenging and even more when you have a thousand
+people's opinion about it.
+
+It's the board's job to `set course every six months`_ (or even less)
+on what we should be focusing on for the next six months.
+
+When this happens, it often means changing our course and killing
+projects or at least putting them on the side if teams are turning to
+new priorities.
+
+.. _`set course every six months`: https://air.mozilla.org/mozilla-all-hands-hawaii-2016-20161221/
+
+.. |OKRs| raw:: html
+
+    <abbr title="Objectives and Key results">OKRs</abbr>
+
+How do you define what the priorities are?
+==========================================
+
+.. image:: {filename}/images/true-north.jpg
+    :alt: Mozilla Logo as a Sailing Map — From https://uk.pinterest.com/explore/firefox-logo/
+    :align: center
+
+
+At our team level, we are driven by the use case. We have a vision of
+what we want to build. That vision leads to ideas of feature that we
+are adding to the backlog.
+
+However we are not building anything until we have a proper use case
+for it. As soon as we have a use case it is added to our quaterly
+roadmap.
+
+You end up not spending time on tasks you don't need and for each use
```

Comment:
How about:
"You end up not spending time on tasks you ..." => "It prevents us spending time on tasks we" 
Please use "we" for the rest of the paragraph to be consistent with the previous one as you are still talking about what your team does. Or if you want to give it as a piece of advice then you need to rephrase it a bit